docs/gt4py/conf.py

```diff
@@ -101,7 +101,7 @@
     "special-members": "__call__",
     "undoc-members": True,
 }
-autodoc_mock_imports = []
+autodoc_mock_imports = ["dace"]
```

Comment:
@egparedes any better ways to make this work?

Reply:
I don't think so. It looks to me like this is the proper way to make it work